addon/actions.js

```diff
@@ -26,10 +26,90 @@ export function updateValidationResults (validationResult) {
   }
 }
 
+function schemaFromRef (definitions) {
+  function invalidPath (refPath) {
```

Comment:
It seems like it would make more sense to move this function outside of its parent function.